client/reader/stream/featured-asset.jsx

```diff
@@ -0,0 +1,57 @@
+/**
+* External dependencies
+*/
+import React from 'react';
+
+export default class FeaturedAsset extends React.Component {
+	constructor( props ) {
+		super( props );
+		this.state = {
+			featuredImage: '',
+			useFeaturedEmbed: props.useFeaturedEmbed
+		};
+	}
+
+	componentDidMount() {
+		if ( this.props.featuredImage ) {
```

Comment:
How does this look https://github.com/Automattic/wp-calypso/pull/8201/commits/5966d678b48973e85fccc98c75a2f19f2d58b048 ?